fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalItemTags.java

```diff
@@ -308,6 +308,24 @@ private ConventionalItemTags() {
 	public static final TagKey<Item> UNCOLORED_SANDSTONE_SLABS = register("sandstone/uncolored_slabs");
 	public static final TagKey<Item> UNCOLORED_SANDSTONE_STAIRS = register("sandstone/uncolored_stairs");
 
+	// Flower
+	/**
+	 * Contains living ground-based flowers that are 1 block tall such as Dandelions or Poppy.
+	 * Equivalent to the "minecraft:small_flowers" item tag.
```

Comment:
```suggestion
	 * Equivalent to the {@code minecraft:small_flowers} item tag.
```
Code formatting looks nicer for ids